app/models/miq_server/log_management.rb

```diff
@@ -109,7 +109,12 @@ def post_one_log_pattern(pattern, logfile, log_type)
     task = logfile.miq_task
     log_prefix = "Task: [#{task.id}]"
 
-    log_start, log_end = log_start_and_end_for_pattern(pattern)
+    if %w[Dialogs Models].include?(log_type)
```

Comment:
It looks like the logging_started_on is blank for non-dialog/models calls... can we do this?

```ruby
log_start, log_end = if logfile.logging_started_on
   logfile.logging_started_on, logfile.logging_ended_on
else
  log_start_and_end_for_pattern(pattern)
end
```